openghg/store/storage/_chunking.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+from typing import Dict, List
+
+
+@dataclass(frozen=True)
+class ChunkingSchema:
+    """Chunking scheme for zarr storage.
+
+    Args:
+        variable: Name of the variable
+        chunks: Dictionary of chunk sizes for each dimension
+        secondary_dimensions: List of secondary dimensions to chunk over
+    """
+
+    # TODO - open to alternative names here
+
+    variable: str
+    chunks: Dict[str, int]
+    secondary_dims: List[str]
+    # max chunk size in bytes (250 MB)
+    # TODO - do we want to make this configurable?
+    max_chunk_size: int = 262144000
```

Comment:
As Brendan mentioned - we may need to up this to the equivalent of 300MB to be able to manage high time resolution footprint chunks.

Reply:
Done.